lib/std/os/linux.zig

```diff
@@ -3536,85 +3536,86 @@ pub const dl_phdr_info = extern struct {
-pub const CPU_SETSIZE = 1024;
-pub const cpu_set_t = [CPU_SETSIZE / @sizeOf(usize)]usize;
-
 /// the n-th cpu bit is at ELT: n / M, BIT: n % M
 /// if n-th bit is 1, the n-th cpu is in the cpu_set, otherwise it is not
 const cpu_shift = std.math.log2(@sizeOf(usize));
 const cpu_shift_t = std.meta.Int(.unsigned, cpu_shift);
-inline fn cpu_elt(cpu: usize) usize {
-    // when M is 2^k, n / M <=> n >> k
+inline fn cpu_elt(cpu: cpu_count_t) usize {
+    // M is 2^k, so n / M <=> n >> k
     return cpu >> cpu_shift;
 }
-inline fn cpu_mask(cpu: usize) usize {
-    // when M is 2^k, n % M <=> n & (M - 1)
-    var x = cpu & (@sizeOf(usize) - 1);
-    return @as(usize, 1) << @intCast(cpu_shift_t, x);
+inline fn cpu_mask(cpu: cpu_count_t) usize {
+    // M is 2^k, so n % M <=> n & (M - 1)
+    var x: cpu_shift_t = cpu & (@sizeOf(usize) - 1);
+    return 1 << x;
 }
 
 pub fn CPU_ZERO(set: *cpu_set_t) void {
-    @memset(set, 0);
-}
-
-pub fn CPU_SET(cpu: usize, set: *cpu_set_t) void {
-    if (cpu >= CPU_SETSIZE) {
-        return;
-    }
-    (set.*)[cpu_elt(cpu)] |= cpu_mask(cpu);
-}
-pub fn CPU_CLR(cpu: usize, set: *cpu_set_t) void {
-    if (cpu >= CPU_SETSIZE) {
-        return;
-    }
-    (set.*)[cpu_elt(cpu)] &= ~cpu_mask(cpu);
-}
-pub fn CPU_ISSET(cpu: usize, set: cpu_set_t) bool {
-    if (cpu >= CPU_SETSIZE) {
-        return false;
-    }
-    return set[cpu_elt(cpu)] & cpu_mask(cpu) != 0;
-}
-
-pub fn CPU_COUNT(set: cpu_set_t) usize {
-    var sum: usize = 0;
+     @memset(set, 0);
+ }
+
+ pub fn CPU_SET(cpu: usize, set: *cpu_set_t) void {
+     if (cpu >= CPU_SETSIZE) {
+         return;
+     }
+     (set.*)[cpu_elt(cpu)] |= cpu_mask(cpu);
+ }
+ pub fn CPU_CLR(cpu: usize, set: *cpu_set_t) void {
+     if (cpu >= CPU_SETSIZE) {
+         return;
+     }
+     (set.*)[cpu_elt(cpu)] &= ~cpu_mask(cpu);
+ }
+ pub fn CPU_ISSET(cpu: usize, set: cpu_set_t) bool {
+     if (cpu >= CPU_SETSIZE) {
+         return false;
+     }
+     return set[cpu_elt(cpu)] & cpu_mask(cpu) != 0;
+ }
+
+pub fn CPU_COUNT(set: cpu_set_t) cpu_count_t {
+    var sum: cpu_count_t = 0;
     for (set) |x| {
         sum += @popCount(x);
     }
     return sum;
 }
 
-pub fn CPU_AND(destset: *cpu_set_t, srcset1: cpu_set_t, srcset2: cpu_set_t) void {
-    for (destset, srcset1, srcset2) |*dst, a, b| {
-        dst.* = a & b;
-    }
-}
-pub fn CPU_OR(destset: *cpu_set_t, srcset1: cpu_set_t, srcset2: cpu_set_t) void {
-    for (destset, srcset1, srcset2) |*dst, a, b| {
-        dst.* = a | b;
-    }
-}
-pub fn CPU_XOR(destset: *cpu_set_t, srcset1: cpu_set_t, srcset2: cpu_set_t) void {
-    for (destset, srcset1, srcset2) |*dst, a, b| {
-        dst.* = a ^ b;
-    }
-}
-
-pub fn CPU_EQUAL(srcset1: cpu_set_t, srcset2: cpu_set_t) bool {
-    for (srcset1, srcset2) |a, b| {
-        if (a != b) {
-            return false;
-        }
-    }
-    return true;
-}
+ pub fn CPU_AND(destset: *cpu_set_t, srcset1: cpu_set_t, srcset2: cpu_set_t) void {
```

Comment:
Please run zig fmt and report shortly, if tests worked for you locally. We can then ping a core member to rerun CI or wait (if you are ok with it).

Reply:
Thank you for the quick remind, I am fixing it